openssl-dynamic/src/main/c/jnilib.c

```diff
@@ -418,13 +426,6 @@ jint JNI_OnLoad_netty_tcnative(JavaVM* vm, void* reserved) {
         dllPath[dllPathLen] = '\0';
     }
 
-    char* dllTmpPath = dllPath;
```

Comment:
discussed offline with @normanmaurer ... I think getting rid of this path manipulation should be enough.